<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_SimpleDriverBackup</class>
 <widget class="QWidget" name="Widget_SimpleDriverBackup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget_SimpleDriverBackup</string>
  </property>
  <widget class="QLabel" name="label_help">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>341</width>
     <height>93</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 255, 194);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;

The backup process will copy all subfolders and files in&lt;br&gt;
&lt;span style=&quot; font-style:italic;&quot;&gt;%windir%\system32\DriverStore&lt;/span&gt; to your requested destination.&lt;br&gt;&lt;br&gt;

Reinstalling the drivers can easily be achieved by using the builtin &lt;br&gt;
&lt;span style=&quot; font-style:italic;&quot;&gt;Device Manager&lt;/span&gt; in Windows and pointing to the driver backup folder.

&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_destFolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_currentFileCopying">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>File:</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_destFolder">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>126</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select destination folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>126</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start backup</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_about">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>80</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Secret button</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_title">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Simple Driver Backup&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_version">
      <property name="text">
       <string>version </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>126</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pixHDD">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>111</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
